src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -69,6 +69,11 @@
   private LanguageMode languageOut;
 
   /**
+   * If true, allows experimental ES6 to ES6 compilation.
+   */
+  boolean allowEs6ToEs6;
```

Comment:
Let's make this private with a getter and setter. I know it feels overkill but it may actually make things easier if we need to refactor.